.travis.yml

```diff
@@ -26,98 +26,86 @@ branches:
   only:
     - master
 
+# TODO: Test on the minimum and maximum supported platform versions
+
 jobs:
   include:
     - stage: Node tests
-      osx_image: xcode10.2
+      osx_image: xcode10
       language: node_js
       node_js: "10"
       install: npm install
       script: npm run test
 
     - stage: WDA build
-      name: iPhone SE, Xcode 9.2
-      osx_image: xcode9.2
-      env: IPHONE_MODEL="iPhone SE" IPAD_MODEL="iPad Air 2" IOS_VERSION="11.2" ACTION=build TARGET=runner
-    - name: iPhone X, Xcode 10.2
-      osx_image: xcode10.2
-      env: IPHONE_MODEL="iPhone X" IPAD_MODEL="iPad Air 2" IOS_VERSION="12.2" ACTION=build TARGET=runner
-    - name: apple tv, Xcode 10.2
-      osx_image: xcode10.2
-      env: DEST=tv TV_MODEL="Apple TV" TV_VERSION="12.2" ACTION=build TARGET=tv_runner SDK=tv_sim
+      # - name: iPhone SE, Xcode 9.2
```

Comment:
just commenting out?

Reply:
yep, wi'll replace them with Xcode 11/iOS 13 as soon as your PR is done

Reply:
👍